model/service_logs/v1/cluster_resource.model

```diff
@@ -0,0 +1,24 @@
+/*
+Copyright (c) 2021 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Manages a specific Cluster for a cluster log.
+resource Cluster {
+
+	// Reference to the list of cluster logs for a specific cluster uuid.
+	locator ClusterLogs {
```

Comment:
This should probably be just `Log`, that way the code to access the logs will look like this:

```go
logsResource := connection.ServiceLogs().V1().Clusters().Cluster("123").Logs()
```

Otherwise it would be like this, which repeats the "cluster" word many times:

```go
logsResource := connection.ServiceLogs().V1().Clusters().Cluster("123").ClusterLogs()
```

Reply:
@jhernand We can change that in the SDK but that won't be consistent with our original path `/api/service_logs/v1/clusters/{uuid}/cluster_logs`

Reply:
OK, don't change it.

Reply:
Note that `cluster_logs` is on purpose because most the paths in OSL end with `cluster_logs` so we add that for consistency. 

For example to list/post logs the path looks like this `/api/service_logs/v1/cluster_logs` so that's why we chose `/api/service_logs/v1/clusters/{uuid}/cluster_logs`

Reply:
I understand, thanks for explaining.